draft-ietf-ohai-ohttp.md

```diff
@@ -1096,11 +1096,12 @@ TLS connections from Clients, so they see message boundaries. This privileged
 position allows for richer feature extraction from encrypted data, which might
 improve traffic analysis.
 
-Clients can use padding to reduce the effectiveness of traffic analysis.
-Padding is a capability provided by binary HTTP messages; see {{Section 3.8 of
-BINARY}}.  If the encapsulation method described in this document is used to
-protect a different message type (see {{repurposing}}), that message format
-might need to include padding support.
+Clients and Oblivious Gateway Resources can use padding to reduce the
+effectiveness of traffic analysis.  Padding is a capability provided by binary
+HTTP messages; see {{Section 3.8 of BINARY}}.  If the encapsulation method
+described in this document is used to protect a different message type (see
+{{repurposing}}), that message format might need to include padding support.
```

Comment:
```suggestion
Clients can use padding to reduce the effectiveness of traffic analysis.
Client padding is a capability provided by binary HTTP messages; see
{{Section 3.8 of BINARY}}.  If the encapsulation method described in this
document is used to protect a different message type (see {{repurposing}}),
that message format might need to include padding support. Oblivious Gateway
Resources can also use padding for the same reason, though the mechanism
for doing so is via header or trailer fields.
```

The current text makes it seem like the gateway pads via BHTTP, which isn't possible. So in this suggested text I tried to make it clear that the gateway can also pad, but it requires a separate mechanism.

Reply:
Isn't it possible for the gateway to pad responses?

Reply:
I'm on the edge about the relay.  There are definitely advantages in terms of traffic analysis resistance, so I think we can handle the offhand mention.

Reply:
On consideration, I'm going to make a suggestion for the relay in a separate PR.